controllers/provider-alicloud/pkg/webhook/controlplanebackup/ensurer.go

```diff
@@ -64,6 +75,15 @@ func (e *ensurer) ensureContainers(ps *corev1.PodSpec, name string, cluster *ext
 	return nil
 }
 
+func (e *ensurer) ensureChecksumAnnotations(ctx context.Context, template *corev1.PodTemplateSpec, namespace, name string) error {
+	if name == common.EtcdMainStatefulSetName {
+		if err := controlplane.EnsureSecretChecksumAnnotation(ctx, template, e.client, namespace, alicloud.BackupSecretName); err != nil {
```

Comment:
```suggestion
		return controlplane.EnsureSecretChecksumAnnotation(ctx, template, e.client, namespace, alicloud.BackupSecretName)
```

(replicate for other providers)

Reply:
Done